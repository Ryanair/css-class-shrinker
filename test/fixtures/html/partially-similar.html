<div class="aaaooo"></div>
<div class="eee">
  <h1 class="iii aaa-ooo-uuu"></h1>
  <button class="ooo"></button>
  <div class="uuu qqq-eee"></div>
</div>
