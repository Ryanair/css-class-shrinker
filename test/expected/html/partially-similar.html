<div class="aaaooo"></div>
<div class="lpdD">
  <h1 class="fsG aaa-ooo-uuu"></h1>
  <button class="acvD"></button>
  <div class="gDs qqq-eee"></div>
</div>
